business_requirement_deliverable/views/business_view.xml

```diff
@@ -31,14 +31,14 @@
             <field name="model">business.requirement.deliverable</field>
             <field name="arch" type="xml">
                 <tree string="Business Requirement Deliverable" default_order="sequence">
-                    <field name="currency_id" invisible="1"/>
+                    <field name="currency_id" invisible="1" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
```

Comment:
@elicoidal I think @pedrobaeza is right

The meaning to add parameter `groups` into the view is to make this field only visible for those groups listed is not like a real ACL.

So actually I would ask @sudhir-serpentcs to remove parameter invisible so now will make sense.